ch-algorithms/quantum-fourier-transform.ipynb

```diff
@@ -633,7 +633,8 @@
    ],
    "source": [
     "# Use the tenerife (ibmqx4) device\n",
-    "backend = IBMQ.get_backend('ibmqx4')\n",
+    "IBMQ.get_provider(hub='ibm-q')\n",
+    "backend = provider.get_backend('ibmqx4')\n",
```

Comment:
Can you please replace this one with `ibmq_vigo`? `ibmqx4` is no longer available.